lms/djangoapps/grades/tasks.py

```diff
@@ -10,17 +10,17 @@
 from lms.djangoapps.course_blocks.api import get_course_blocks
 from opaque_keys.edx.keys import UsageKey
 from opaque_keys.edx.locator import CourseLocator
-from openedx.core.djangoapps.content.block_structure.api import get_course_in_cache
+from openedx.core.djangoapps.performance.utils import collect_profile_func
 from xmodule.modulestore.django import modulestore
 
 from .config.models import PersistentGradesEnabledFlag
-from .new.course_grade import CourseGradeFactory
 from .new.subsection_grade import SubsectionGradeFactory
 from .signals.signals import SUBSECTION_SCORE_CHANGED
 from .transformer import GradesTransformer
 
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
+@collect_profile_func('recalculate_subsection_grade', enabled=False)
```

Comment:
True - I should delete that line since it doesn't serve any purpose to keep there.  I was thinking if we had an automated profiling test framework, we could potentially use this decorator to enable it for certain tests and not for production.  Anyway, I've kept the implementation of the decorator in a new performance/utils file so we can easily use it again.  But I'll go ahead and delete its use of it in the code.